app/facades/friendship_facade.rb

```diff
@@ -0,0 +1,33 @@
+class FriendshipFacade
+  def self.get_friends(user_id)
+    friends = FriendshipService.get_friends(user_id)
+    return [] unless friends[:data]
+
+    friends[:data].map { |friend| User.new(friend) }
+  end
+
+  def self.get_friends_at_gym(yelp_gym_id, current_user_id)
+    friends = FriendshipService.get_friends_at_gym(yelp_gym_id, current_user_id)
+    return [] unless friends[:data]
+
+    friends[:data].map { |friend| User.new(friend) }
+  end
+
+  def self.get_non_friends_at_gym(params)
+    non_friends = FriendshipService.get_non_friends_at_gym(params)
+    return unless non_friends[:data]
```

Comment:
Any reason we are not returning empty array in this guard clause, like the `get_friends_at_gym` and `get_friends` methods?

Reply:
Good catch!  I agree that we should return an empty array here.  I'll track this one down.